src/App.js

```diff
@@ -1,27 +1,27 @@
-import logo from './logo.svg';
+import Header from './components/Header/Header';
+import Slideshow from './components/Slideshow/Slideshow';
+import Categories from './components/Categories/Categories';
+import Products from './components/Products/Products';
+import Footer from './components/Footer/Footer';
 import './App.css';
+import mock1 from './assets/mocks/es-mx/featured-banners.json';
+import mock2 from './assets/mocks/en-us/product-categories.json';
+import mock3 from './assets/mocks/en-us/featured-products.json';
 import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
 
 function App() {
   const { data, isLoading } = useFeaturedBanners();
   console.log(data, isLoading);
```

Comment:
if this console.log is not required, delete it. In almost all code implementation that is going to be merged, the code should avoid console.logs on prod.

Reply:
This sonsole.log came by default in the project when I forked it, I forgot to remove it. I'm on it. Thank you!